src/rpc/blockchain.cpp

```diff
@@ -418,6 +420,16 @@ void entryToJSON(UniValue &info, const CTxMemPoolEntry &e)
     }
 
     info.pushKV("spentby", spent);
+
+    // Add opt-in RBF status
+    std::string rbfStatus = "no";
+    RBFTransactionState rbfState = IsRBFOptIn(tx, mempool);
+    if (rbfState == RBF_TRANSACTIONSTATE_UNKNOWN)
```

Comment:
please use brackets for multi-line conditionals

Reply:
Thanks, updated.